--- wxHexEditor/DataInterpreter.h.org	2009-08-28 01:50:53.000000000 +0200
+++ wxHexEditor/DataInterpreter.h	2009-08-28 01:51:17.000000000 +0200
@@ -32,6 +32,8 @@
 #ifndef DATAINTERPRETER_H
 #define DATAINTERPRETER_H
 
+#include <stdint.h>
+
 class DataInterpreter : public InterpreterGui{
 	protected:
 		struct unidata{
--- wxHexEditor/HexEditorCtrl/wxHexCtrl/Tag.h.org	2009-08-28 01:54:32.000000000 +0200
+++ wxHexEditor/HexEditorCtrl/wxHexCtrl/Tag.h	2009-08-28 01:54:56.000000000 +0200
@@ -28,6 +28,7 @@
 #include <wx/popupwin.h>
 #include <wx/dcmemory.h>
 #include "TagDialogGui.h"
+#include <stdint.h>
 
 class TagElement{
 	public:
